app/views/posts/_expanded.html.erb

```diff
@@ -293,6 +293,33 @@
           </div>
         <% end %>
 
+        <% if is_question && post.children.length >= SiteSetting["TableOfContentsThreshold"] %>
+          <div class="toc" id="toc-example-toggle">
+            <button class="toc--header" data-toggle="#toc-example-toggle" data-toggle-property="class" data-toggle-value="is-active">Table of Contents</button>
+            <% post.children.each do |answer| %>
```

Comment:
[**praise:**](https://conventionalcomments.org/) Good idea to use post.children instead of generating a new database query in the controller (which was my original idea TBF). This is definitely cleaner and faster.